main.tsp

```diff
@@ -14,6 +14,7 @@ import "./messages";
 import "./models";
 import "./moderations";
 import "./runs";
+import "./streaming";
```

Comment:
Could you update the openapi.yaml file at the root of the repo? It's just a copy of OpenAI's openapi.yaml found here:
https://github.com/openai/openai-openapi/blob/master/openapi.yaml

And then, the README.md at the root of the repo also has a permalink to the "snapshot" of the openapi.yaml with the corresponding commit hash.